pipelines/LSSTComCamSim/DRP-ops-rehearsal-3.yaml

```diff
@@ -24,6 +26,13 @@ tasks:
       wcs_provider: "input_summary"
       photo_calib_provider: "input_summary"
       background_provider: "input_summary"
+  analyzeObjectTableCore:
+    class: lsst.analysis.tools.tasks.ObjectTableTractAnalysisTask
+    config:
+      python: |
+      # no z-band in comCamSim
+        del config.atools.yPerpPSF
+        del config.atools.yPerpCModel
```

Comment:
I didn't know it was this easy to remove a tool from a task.  Cool! 

I understand why this has to be here and not obs_lsst too: the pipeline yaml overrides get applied after the obs package ones.